src/index.js

```diff
@@ -0,0 +1,52 @@
+import './index.css';
+import displayPokemonList from '../modules/display.js';
+import displayPokemon from '../modules/displayPokemon.js';
+import { likeToPokemon } from '../modules/likesManager.js';
+import { getComments, postComment } from '../modules/involvementRequests.js';
+import displayMessage from '../modules/displayMessage.js';
+import counter from '../modules/cardsCounter.js';
+
+const nameInput = document.querySelector('.name-form');
+const messageInput = document.querySelector('.insights-form');
+const modalPopup = document.querySelector('.modalBackground');
+const closeBtn = document.querySelector('.close');
+
+closeBtn.addEventListener('click', () => {
+  modalPopup.classList.toggle('hidden');
+});
+
+document.querySelector('form').addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const card = JSON.parse(localStorage.getItem('pokemonList'));
+  const index = parseInt(e.target.parentNode.id, 10);
+  // eslint-disable-next-line no-empty
+  await postComment(card[index].data.id, nameInput.value, messageInput.value);
+  const b = await getComments(card[index].data.id);
+  displayMessage(b);
+  // eslint-disable-next-line no-const-assign
+  nameInput.value = '';
+  // eslint-disable-next-line no-const-assign
+  messageInput.value = '';
```

Comment:
Please don't disable the linters. Consider fixing the linter issues instead of disabling the linters.